mitodl/micromasters
# Upgrade glob-parent to 5.1.2 through webpack and babel-plugin-istanbul

#### Pre-Flight checklist


#### What are the relevant tickets?
#4956 

#### What's this PR do?
- Upgrades all the `glob-parent` versions to use from  `5.1.2` in order to fix the alert
- Adds a resolution for `chokidar": ^3.4.3` only for `watchpack/webpack-chokidar2`, if we don't add this resolution we'll have to migrate the whole `webpack` from 4 to 5.

#### How should this be manually tested?
- Check the CI
- Check the webpack build is successful
- Good to run and go through the ap in general
